File: Goorm/Introduction-To-Algorithm/Chapter01/problem01B.cpp
#include <stdio.h>
#include <iostream>
using namespace std;

int getMax(int data[], int n) {
  int answer = data[0];
  for(int i=1 ; i<n ; i++) {
    if (data[i] > answer)
      answer = data[i];
  }
  return answer;
}

int main() {
  int n;
  int * data;

  scanf("%d", &n);
  data = new int[n];

  for(int i=0 ; i<n ; i++) {
    scanf("%d", &data[i]);
  }

  int answer = getMax(data, n);
  printf("%d\n", answer);

  delete[] data;

  return 0;
}

File: Goorm/Introduction-To-Algorithm/Chapter01/problem01B.py
def get_max(data, n):
  max_num = data[0]
  for value in data:
    if value > max_num:
      max_num = value
  return max_num

if __name__ == '__main__':
  n = int(input())
  data = list(map(int, input().split()))
  # data = [int(word) for word in input().split()]
  answer = get_max(data, n)
  print(answer)